WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -601,6 +613,11 @@ def my_competitions
                               .where(id: competition_ids.uniq)
                               .sort_by { |comp| comp.start_date || Date.today + 20.year }.reverse
     @past_competitions, @not_past_competitions = competitions.partition(&:is_probably_over?)
+    bookmarked_ids = current_user.competitions_bookmarked.pluck(:competition_id)
+    @bookmarked_competitions = Competition.includes(:delegate_report, :delegates)
+                                          .not_over
+                                          .where(id: bookmarked_ids.uniq)
+                                          .sort_by { |comp| comp.start_date }
```

Comment:
Can't this be shortened to `.sort_by(&:start_date)`?

Also, are the includes above used for something?

Reply:
Whoops, bad copy/paste.  Done and done.

Reply:
Hmm, I see `.sort_by { :start_date }` instead of `.sort_by(&:start_date)`, any particular reason for that? I quickly checked and can't find this format used elsewhere, but I may be wrong.